library/cmake/Modules/FindPHP.cmake

```diff
@@ -87,26 +101,30 @@ MARK_AS_ADVANCED(
 	PHP_VERSION_STRING
 )
 
+set(development_component_accpeted TRUE)
```

Comment:
As for all internal variables, I would prefix this one with PHP and use uppercase: PHP_DEVELOPMENT_COMPONENT_ACCEPTED

Reply:
Good idea -- I had a strange reason for making it lowercase, but I forgot it anyways, so...derp!  😛